src/exports/roo-code.d.ts

```diff
@@ -261,11 +261,12 @@ type GlobalSettings = {
 				roleDefinition: string
 				customInstructions?: string | undefined
 				groups: (
-					| ("read" | "edit" | "browser" | "command" | "mcp" | "modes")
+					| ("read" | "edit" | "browser" | "command" | "mcp" | "subtask" | "switch" | "followup")
 					| [
-							"read" | "edit" | "browser" | "command" | "mcp" | "modes",
+							"read" | "edit" | "browser" | "command" | "mcp" | "subtask" | "switch" | "followup",
 							{
 								fileRegex?: string | undefined
+								slugRegex?: string | undefined
```

Comment:
Let me see if I change my mind after reading the whole diff, but I think an array of slugs might make more sense than a regex here.

Reply:
Yeah, still think an array of slugs would be easier to manage.